### ISSUE TITLE
Ability to add tooltips

### ISSUE THREAD

**Issue by ievgenkainov** [YUzhva/react-leaflet-markercluster#7]
Hi. Thanks for the implementation beforehand.
I would like to use your lib but faced a problem with tooltips.

With react-leaflet i am using tooltips as:
```
import { Marker, Tooltip } from 'react-leaflet';

<Marker
    icon={someIcon}
    position={somePosition}
    onClick={someOnClickHandler}
  >
    <Tooltip
      permanent
      direction="top"
      offset={point(0, -20)}
    >
      <span>Tooltip text</span>
    </Tooltip>
</Marker>

```

So a tooltip is a child of Marker element. I have a collection of such markers, that i render on the map.

In case of `react-leaflet-markercluster`, MarkerClusterGroup expects array of objects, and there is only one possibility to add something else and it is a popup.

Would be cool if either the lib allows to add tooltips with `tooltip` option for each markers, the same way as implemented for popups, or not only an array of objects (markers' config) but also an array of Markers are accepted.

**Comment by YUzhva** [comment]
What do you think about:

```javascript
const markers = [
    {lat: 49.8397, lng: 24.0297}, tooltip: "my tooltip text",
    {lat: 52.2297, lng: 21.0122},
    {lat: 51.5074, lng: -0.0901}
  ];
```

```javascript
const toolpitWithOptions = L.tooltip({direction: 'bottom'})
    .setContent('ANOTHER TEXT');

const markers = [
    {lat: 49.8397, lng: 24.0297},
    {lat: 52.2297, lng: 21.0122},
    {lat: 51.5074, lng: -0.0901}, toolpit: toolpitWithOptions, 
  ];
```

**Comment by jackkainov** [comment]
Looks good 👍 . Should i try to create a pull request or you have time to do such an enhancement?

**Comment by YUzhva** [comment]
@ievgenkainov 

Implemented in: [release] 1.0.9: Marker tooltip support (#8)

Usage example:
https://yuzhva.github.io/react-leaflet-markercluster/#marker-tooltip

**Comment by jackkainov** [comment]
Awesome, thanks. Was doing the same implementation :)
